src/randomness.rs

```diff
@@ -3,54 +3,64 @@ use crate::errors::NestedSTARError;
 use crate::format::RandomnessSampling;
 use ppoprf::ppoprf;
 
-use reqwest::blocking::Client as HttpClient;
-use serde::{Deserialize, Serialize};
-
-const QUERY_LABEL: &str = "Client randomness request";
-const RESPONSE_LABEL: &str = "Server randomness response";
-
-/// Explicit request body.
-#[derive(Serialize, Clone)]
-pub struct Request {
-  name: String,
-  epoch: u8,
-  points: Vec<ppoprf::Point>,
-}
-
-/// Explicit response body.
-#[derive(Serialize, Deserialize)]
-pub struct Response {
-  name: String,
-  results: Vec<ppoprf::Evaluation>,
+pub fn process_randomness_response(
+  points: &[ppoprf::Point],
+  resp_points: &[&[u8]],
```

Comment:
This function could also just take the full `eval` response directly.

Reply:
If we agree that JSON parsing is the responsibility of the caller, this should not apply